### DISCUSSION TITLE
How to prevent the user from adding formats other than images?

### DISCUSSION THREAD

**Discussion by lucabfelix** [inventare/django-image-uploader-widget#198] [Ideas]
Hello, first of all, thank you for this great image uploader app for Django.

I would like to know if there is a way to prevent the user from uploading any file format other than images.

The second idea is if it would be possible to implement a line break for the ArrayField? When there are many images to be uploaded, the single-line experience starts to become less pleasant for the user.

**Comment by EduardoJM** [comment]
Thanks for your question

> I would like to know if there is a way to prevent the user from uploading any file format other than images.

When using `ImageUploaderWidget` and `ImageUploaderInline` is possible to change the accept: [docs for widget](https://inventare.github.io/django-image-uploader-widget/widget/04-accept/) and [docs for inline](https://inventare.github.io/django-image-uploader-widget/inline_admin/03-accept/). But for ArrayField, when using `ImageListField` it was not planed and not developed. I opened, now, a issue to add support for this (#199).

> The second idea is if it would be possible to implement a line break for the ArrayField? When there are many images to be uploaded, the single-line experience starts to become less pleasant for the user.

Is a good idea create a variant stylesheet to show the images as a grid instead of horizontal scroll. I opened a new issue for that too (#200).

**Comment by EduardoJM** [comment]
The fix for the accept problem was released into version 1.0.1 at PyPI.
